### ISSUE TITLE
Package.activateConfig is deprecated.

### ISSUE THREAD

**Issue by madhon** [yoshiori/linter-golint#2]
Use a config schema instead. See the configuration section
of https://atom.io/docs/latest/hacking-atom-package-word-count and
https://atom.io/docs/api/latest/Config for more details
```
Package.activateConfig (C:\Users\user\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:267:11)
Package.activateNow (C:\Users\user\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:238:14)
<unknown> (C:\Users\jdouglas\user\Local\atom\app-0.198.0\resources\app.asar\src\package.js:223:30)
Package.measure (C:\Users\juser\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:167:15)
```

**Comment by kyoh86** [comment]
+1

**Comment by jordandm** [comment]
+1

**Comment by idotpi** [comment]
linter-golint (2 deprecations)
Check for UpdateDisable Package
Use a config schema instead. See the configuration section of https://atom.io/docs/latest/hacking-atom-package-word-count and https://atom.io/docs/api/latest/Config for more details
Report Issue
Package.activateConfig - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:265:11
Package.activateNow - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:236:14
<unknown> - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:221:30
Package.measure - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:165:15
Package.activate - /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:213:14
PackageManager.activatePackage - /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:452:21

**Comment by yoshiori** [comment]
fix bd059260e711c8f254de0592a94571ae3a6eaef5
